### PR TITLE
Spawn number

### LINKED ISSUE #29: Make spawn number configurable

### PR DESCRIPTION
**Pull request by 1-alex98**
Fixes #29

### CODE REVIEW

File: src/java/generator/MapGenerator.java [hunk 1]
```diff
@@ -76,10 +77,104 @@ public void save(String folderName, String mapName, SCMap map, long seed) {
 		}
 	}
 
+	private static void interpretArguments(String[] args) {
```
Comment (Geosearchef) [inline src/java/generator/MapGenerator.java]: Why did you move this entire block? It's probably better to have the generation at the top.
This also makes it impossible to see what parts you changed, please move it back.

**Review by Geosearchef — CHANGES_REQUESTED** [review]

File: src/java/generator/MapGenerator.java (hunk 1 above)
Comment (1-alex98) [reply src/java/generator/MapGenerator.java]: used intelij to structure code

**Comment by n3roxis** [comment]
Seems to work well. It should be noted that increasing the spawn count will cause the generation to fail in rare cases, since there wont be enough space to put the spawns. I'll add an issue for this.

File: src/java/generator/MapGenerator.java (hunk 1 above)
Comment (n3roxis) [reply src/java/generator/MapGenerator.java]: @axel1200 please resolve this so I can merge your additions

**Comment by 1-alex98** [comment]
I linearly decreased the separation of spawns so, maybe it works always. At least in my tests it worked :D

**Comment by n3roxis** [comment]
Alright, I'll merge this then (when geos request is resolved)

**Review by n3roxis — APPROVED** [review]

**Comment by 1-alex98** [comment]
I guess let's not merge @shalkya wants to build a different slution on how to pass these arumnets I guess

**Comment by shalkya** [comment]
@axel1200 i'm keyser

**Comment by Katharsas** [comment]
axel meant @Sheikah45
